web/src/components/__tests__/CleanLink.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import CleanLink from '../CleanLink'
+import { renderWithRouter } from '../../testing/render'
+import { render } from '@testing-library/react'
+
+describe('CleanLink', () => {
+  const linkInternal = '/augsburg/de/willkommen/willkommen-in-augsburg'
+  const externalLink = 'https://integreat.app/'
```

Comment:
Shouldn't this be an internal link to the root of integreat, i.e.to the path `/`?

Reply:
indeed it was a bad example ;)